stock_checkout_sync/tests/test_checkout_sync.py

```diff
@@ -66,6 +66,23 @@ def setUpClass(cls):
         cls.picking_pack = cls.pack_move1.picking_id
         cls.picking_pack_post = cls.pack_move4.picking_id
 
+    def _prepare_pickings(self, with_check=True):
+        self.pack_type.checkout_sync = True
+        self.pack_post_type.checkout_sync = True
+
+        self.moves._action_assign()
+        if not with_check:
```

Comment:
with_check is always False when called from tests... what is for?

Reply:
Thanks @simahawk for your cmment. I did hesistate to update other test. But now the PR is updated.